server/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java

```diff
@@ -143,25 +143,22 @@ private Session session() {
         return session;
     }
 
-    public CompletableFuture<Long> executeAlterTableOpenClose(List<RelationName> tables,
+    public CompletableFuture<Long> executeAlterTableOpenClose(DocTableInfo tableInfo,
                                                               boolean openTable,
-                                                              boolean ignoreUnavailableIndices,
                                                               @Nullable PartitionName partitionName) {
         String partitionIndexName = null;
         if (partitionName != null) {
             partitionIndexName = partitionName.asIndexName();
         }
         if (openTable || clusterService.state().getNodes().getMinNodeVersion().before(Version.V_4_3_0)) {
             OpenCloseTableOrPartitionRequest request = new OpenCloseTableOrPartitionRequest(
-                tables,
-                partitionIndexName,
-                openTable,
-                ignoreUnavailableIndices
-            );
+                tableInfo.ident(), partitionIndexName, openTable);
             return transportOpenCloseTableOrPartitionAction.execute(request, r -> -1L);
```

Comment:
`return transportOpenCloseTableOrPartitionAction.execute(request, r -> -1L) `is kept from https://github.com/crate/crate/pull/12280